### ISSUE TITLE
Can't streaming live from the last version (v0.1.18): "Filtering and streamcopy cannot be used together." 

### ISSUE THREAD

**Issue by mrMiimo** [KhaosT/homebridge-camera-ffmpeg#474]
**Describe The Bug:**

Today I updated to the latest version (v0.1.18) and I can't stream live anymore ...
Below I attach my config.json I don't know if there the problem or maybe the changes commit yesterday on ffmpeg.js

Thank you guys! and happy 1 May;)
R.

**Logs:**

```
May 01 19:49:08 raspberrypi homebridge[9407]: ffmpeg -rtsp_transport tcp -re -i rtsp://USERXXX:PWDXXXX@192.168.178.102:554/Streaming/Unicast/channels/102 -map 0:0 -vcodec copy -pix_fmt yuv420p -r 10 -f rawvideo -tune zerolatency -vf scale=1280:720 -b:v 299k -bufsize 299k -maxrate 299k -payload_type 99 -ssrc 13027166 -f rtp -srtp_out_suite AES_CM_128_HMAC_SHA1_80 -srtp_out_params FOQqJt2r+s6J7R98aCfrXQUF9HUcoy/kpZsgdtdX srtp://192.168.178.101:53880?rtcpport=53880&localrtcpport=53880&pkt_size=1316 -loglevel debug
May 01 19:49:09 raspberrypi homebridge[9407]: ffmpeg version 4.1.4-1+rpt1~deb10u1 Copyright (c) 2000-2019 the FFmpeg developers
May 01 19:49:09 raspberrypi homebridge[9407]:   built with gcc 8 (Raspbian 8.3.0-6+rpi1)
May 01 19:49:09 raspberrypi homebridge[9407]:   configuration: --prefix=/usr --extra-version='1+rpt1~deb10u1' --toolchain=hardened --libdir=/usr/lib/arm-linux-gnueabihf --incdir=/usr/include/arm-linux-gnueabihf --arch=arm --enable-gpl --disable-stripping --enable-avresample --disable-filter=resample --enable-avisynth --enable-gnutls --enable-ladspa --enable-libaom --enable-libass --enable-libbluray --enable-libbs2b --enable-libcaca --enable-libcdio --enable-libcodec2 --enable-libflite --enable-libfontconfig --enable-libfreetype --enable-libfribidi --enable-libgme --enable-libgsm --enable-libjack --enable-libmp3lame --enable-libmysofa --enable-libopenjpeg --enable-libopenmpt --enable-libopus --enable-libpulse --enable-librsvg --enable-librubberband --enable-libshine --enable-libsnappy --enable-libsoxr --enable-libspeex --enable-libssh --enable-libtheora --enable-libtwolame --enable-libvidstab --enable-libvorbis --enable-libvpx --enable-libwavpack --enable-libwebp --enable-libx265 --enable-libxml2 --enable-libxvid --enable-libzmq --enable-libzvbi --enable-lv2 --enable-omx --enable-openal --enable-opengl --enable-sdl2 --enable-omx-rpi --enable-mmal --enable-libdc1394 --enable-libdrm --enable-libiec61883 --enable-chromaprint --enable-frei0r --enable-libx264 --enable-shared
May 01 19:49:09 raspberrypi homebridge[9407]:   libavutil      56. 22.100 / 56. 22.100
May 01 19:49:09 raspberrypi homebridge[9407]:   libavcodec     58. 35.100 / 58. 35.100
May 01 19:49:09 raspberrypi homebridge[9407]:   libavformat    58. 20.100 / 58. 20.100
May 01 19:49:09 raspberrypi homebridge[9407]:   libavdevice    58.  5.100 / 58.  5.100
May 01 19:49:09 raspberrypi homebridge[9407]:   libavfilter     7. 40.101 /  7. 40.101
May 01 19:49:09 raspberrypi homebridge[9407]:   libavresample   4.  0.  0 /  4.  0.  0
May 01 19:49:09 raspberrypi homebridge[9407]:   libswscale      5.  3.100 /  5.  3.100
May 01 19:49:09 raspberrypi homebridge[9407]:   libswresample   3.  3.100 /  3.  3.100
May 01 19:49:09 raspberrypi homebridge[9407]:   libpostproc    55.  3.100 / 55.  3.100
May 01 19:49:09 raspberrypi homebridge[9407]: Splitting the commandline.
May 01 19:49:09 raspberrypi homebridge[9407]: Reading option '-rtsp_transport' ...
May 01 19:49:09 raspberrypi homebridge[9407]:  matched as AVOption 'rtsp_transport' with argument 'tcp'.
May 01 19:49:09 raspberrypi homebridge[9407]: Reading option '-re' ... matched as option 're' (read input at native frame rate) with argument '1'.
May 01 19:49:09 raspberrypi homebridge[9407]: Reading option '-i' ... matched as input url with argument 'rtsp://pr0v4!:Pr0v4!23@192.168.178.102:554/Streaming/Unicast/channels/102'.
May 01 19:49:09 raspberrypi homebridge[9407]: Reading option '-map' ... matched as option 'map' (set input stream mapping) with argument '0:0'.
May 01 19:49:09 raspberrypi homebridge[9407]: Reading option '-vcodec' ... matched as option 'vcodec' (force video codec ('copy' to copy stream)) with argument 'copy'.
May 01 19:49:09 raspberrypi homebridge[9407]: Reading option '-pix_fmt' ... matched as option 'pix_fmt' (set pixel format) with argument 'yuv420p'.
May 01 19:49:09 raspberrypi homebridge[9407]: Reading option '-r' ... matched as option 'r' (set frame rate (Hz value, fraction or abbreviation)) with argument '10'.
May 01 19:49:09 raspberrypi homebridge[9407]: Reading option '-f' ... matched as option 'f' (force format) with argument 'rawvideo'.
May 01 19:49:09 raspberrypi homebridge[9407]: Reading option '-tune' ...
May 01 19:49:09 raspberrypi homebridge[9407]:  matched as AVOption 'tune' with argument 'zerolatency'.
May 01 19:49:09 raspberrypi homebridge[9407]: Reading option '-vf' ... matched as option 'vf' (set video filters) with argument 'scale=1280:720'.
May 01 19:49:09 raspberrypi homebridge[9407]: Reading option '-b:v' ... matched as option 'b' (video bitrate (please use -b:v)) with argument '299k'.
May 01 19:49:09 raspberrypi homebridge[9407]: Reading option '-bufsize' ...
May 01 19:49:09 raspberrypi homebridge[9407]:  matched as AVOption 'bufsize' with argument '299k'.
May 01 19:49:09 raspberrypi homebridge[9407]: Reading option '-maxrate' ...
May 01 19:49:09 raspberrypi homebridge[9407]:  matched as AVOption 'maxrate' with argument '299k'.
May 01 19:49:09 raspberrypi homebridge[9407]: Reading option '-payload_type' ...
May 01 19:49:09 raspberrypi homebridge[9407]:  matched as AVOption 'payload_type' with argument '99'.
May 01 19:49:09 raspberrypi homebridge[9407]: Reading option '-ssrc' ...
May 01 19:49:09 raspberrypi homebridge[9407]:  matched as AVOption 'ssrc' with argument '13027166'.
May 01 19:49:09 raspberrypi homebridge[9407]: Reading option '-f' ... matched as option 'f' (force format) with argument 'rtp'.
May 01 19:49:09 raspberrypi homebridge[9407]: Reading option '-srtp_out_suite' ...
May 01 19:49:09 raspberrypi homebridge[9407]:  matched as AVOption 'srtp_out_suite' with argument 'AES_CM_128_HMAC_SHA1_80'.
May 01 19:49:09 raspberrypi homebridge[9407]: Reading option '-srtp_out_params' ...
May 01 19:49:09 raspberrypi homebridge[9407]:  matched as AVOption 'srtp_out_params' with argument 'FOQqJt2r+s6J7R98aCfrXQUF9HUcoy/kpZsgdtdX'.
May 01 19:49:09 raspberrypi homebridge[9407]: Reading option 'srtp://192.168.178.101:53880?rtcpport=53880&localrtcpport=53880&pkt_size=1316' ... matched as output url.
May 01 19:49:09 raspberrypi homebridge[9407]: Reading option '-loglevel' ...
May 01 19:49:09 raspberrypi homebridge[9407]:  matched as option 'loglevel' (set logging level) with argument 'debug'.
May 01 19:49:09 raspberrypi homebridge[9407]: Finished splitting the commandline.
May 01 19:49:09 raspberrypi homebridge[9407]: Parsing a group of options: global .
May 01 19:49:09 raspberrypi homebridge[9407]: Applying option loglevel (set logging level) with argument debug.
May 01 19:49:09 raspberrypi homebridge[9407]: Successfully parsed a group of options.
May 01 19:49:09 raspberrypi homebridge[9407]: Parsing a group of options: input url rtsp://pr0v4!:Pr0v4!23@192.168.178.102:554/Streaming/Unicast/channels/102.
May 01 19:49:09 raspberrypi homebridge[9407]: Applying option re (read input at native frame rate) with argument 1.
May 01 19:49:09 raspberrypi homebridge[9407]: Successfully parsed a group of options.
May 01 19:49:09 raspberrypi homebridge[9407]: Opening an input file: rtsp://pr0v4!:Pr0v4!23@192.168.178.102:554/Streaming/Unicast/channels/102.
May 01 19:49:09 raspberrypi homebridge[9407]: [tcp @ 0xf24460] No default whitelist set
May 01 19:49:09 raspberrypi homebridge[9407]: [tcp @ 0xf24460] Original list of addresses:
May 01 19:49:09 raspberrypi homebridge[9407]: [tcp @ 0xf24460] Address 192.168.178.102 port 554
May 01 19:49:09 raspberrypi homebridge[9407]: [tcp @ 0xf24460] Interleaved list of addresses:
May 01 19:49:09 raspberrypi homebridge[9407]: [tcp @ 0xf24460] Address 192.168.178.102 port 554
May 01 19:49:09 raspberrypi homebridge[9407]: [tcp @ 0xf24460] Starting connection attempt to 192.168.178.102 port 554
May 01 19:49:09 raspberrypi homebridge[9407]: [tcp @ 0xf24460] Successfully connected to 192.168.178.102 port 554
May 01 19:49:09 raspberrypi homebridge[9407]: [rtsp @ 0xf22420] SDP:
May 01 19:49:09 raspberrypi homebridge[9407]: v=0
May 01 19:49:09 raspberrypi homebridge[9407]: o=- 1109162014219182 0 IN IP4 0.0.0.0
May 01 19:49:09 raspberrypi homebridge[9407]: s=Media Server V3.3.4
May 01 19:49:09 raspberrypi homebridge[9407]: i=Media Server Session Description : standard
May 01 19:49:09 raspberrypi homebridge[9407]: e=NONE
May 01 19:49:09 raspberrypi homebridge[9407]: c=IN IP4 0.0.0.0
May 01 19:49:09 raspberrypi homebridge[9407]: t=0 0
May 01 19:49:09 raspberrypi homebridge[9407]: a=control:*
May 01 19:49:09 raspberrypi homebridge[9407]: b=AS:6154
May 01 19:49:09 raspberrypi homebridge[9407]: a=range:npt=now-
May 01 19:49:09 raspberrypi homebridge[9407]: m=video 0 RTP/AVP 96
May 01 19:49:09 raspberrypi homebridge[9407]: i=Video Media
May 01 19:49:09 raspberrypi homebridge[9407]: a=rtpmap:96 H264/90000
May 01 19:49:09 raspberrypi homebridge[9407]: a=fmtp:96 profile-level-id=4D0014;packetization-mode=0;sprop-parameter-sets=Z01AHpplBQF/y/+AXoBfNwEBAUAAAPoAACcQOhgBFcABFbLvLjQwAiuAAitl3lwo,aO48gA==
May 01 19:49:09 raspberrypi homebridge[9407]: a=control:trackID=video
May 01 19:49:09 raspberrypi homebridge[9407]: b=AS:6144
May 01 19:49:09 raspberrypi homebridge[9407]: a=Media_header:MEDIAINFO=494D4B48010100000400000100000000000000000000000000000000000000000000000000000000;
May 01 19:49:09 raspberrypi homebridge[9407]: a=appversion:1.0
May 01 19:49:09 raspberrypi homebridge[9407]: [rtsp @ 0xf22420] video codec set to: h264
May 01 19:49:09 raspberrypi homebridge[9407]: [rtsp @ 0xf22420] RTP Profile IDC: 4d Profile IOP: 0 Level: 14
May 01 19:49:09 raspberrypi homebridge[9407]: [rtsp @ 0xf22420] RTP Packetization Mode: 0
May 01 19:49:09 raspberrypi homebridge[9407]: [rtsp @ 0xf22420] Extradata set to 0xf25ee0 (size: 60)
May 01 19:49:09 raspberrypi homebridge[9407]: [rtsp @ 0xf22420] setting jitter buffer size to 0
May 01 19:49:09 raspberrypi homebridge[9407]: [rtsp @ 0xf22420] hello state=0
May 01 19:49:09 raspberrypi homebridge[9407]: [h264 @ 0xf25a70] nal_unit_type: 7(SPS), nal_ref_idc: 3
May 01 19:49:09 raspberrypi homebridge[9407]: [h264 @ 0xf25a70] nal_unit_type: 8(PPS), nal_ref_idc: 3
May 01 19:49:09 raspberrypi homebridge[9407]: [h264 @ 0xf25a70] nal_unit_type: 7(SPS), nal_ref_idc: 3
May 01 19:49:09 raspberrypi homebridge[9407]: [h264 @ 0xf25a70] nal_unit_type: 8(PPS), nal_ref_idc: 3
May 01 19:49:09 raspberrypi homebridge[9407]: [h264 @ 0xf25a70] nal_unit_type: 1(Coded slice of a non-IDR picture), nal_ref_idc: 3
May 01 19:49:09 raspberrypi homebridge[9407]: [h264 @ 0xf25a70] Format yuvj420p chosen by get_format().
May 01 19:49:09 raspberrypi homebridge[9407]: [h264 @ 0xf25a70] Reinit context to 640x368, pix_fmt: yuvj420p
May 01 19:49:09 raspberrypi homebridge[9407]: [h264 @ 0xf25a70] Frame num gap 32 30
May 01 19:49:09 raspberrypi homebridge[9407]: [h264 @ 0xf25a70] nal_unit_type: 1(Coded slice of a non-IDR picture), nal_ref_idc: 3
May 01 19:49:10 raspberrypi homebridge[9407]:     Last message repeated 16 times
May 01 19:49:10 raspberrypi homebridge[9407]: [h264 @ 0xf25a70] nal_unit_type: 7(SPS), nal_ref_idc: 3
May 01 19:49:10 raspberrypi homebridge[9407]: [h264 @ 0xf25a70] nal_unit_type: 8(PPS), nal_ref_idc: 3
May 01 19:49:10 raspberrypi homebridge[9407]: [h264 @ 0xf25a70] nal_unit_type: 5(IDR), nal_ref_idc: 3
May 01 19:49:10 raspberrypi homebridge[9407]: [h264 @ 0xf25a70] nal_unit_type: 1(Coded slice of a non-IDR picture), nal_ref_idc: 3
May 01 19:49:11 raspberrypi homebridge[9407]:     Last message repeated 5 times
May 01 19:49:11 raspberrypi homebridge[9407]: [rtsp @ 0xf22420] All info found
May 01 19:49:11 raspberrypi homebridge[9407]: Input #0, rtsp, from 'rtsp://pr0v4!:Pr0v4!23@192.168.178.102:554/Streaming/Unicast/channels/102':
May 01 19:49:11 raspberrypi homebridge[9407]:   Metadata:
May 01 19:49:11 raspberrypi homebridge[9407]:     title           : Media Server V3.3.4
May 01 19:49:11 raspberrypi homebridge[9407]:     comment         : Media Server Session Description : standard
May 01 19:49:11 raspberrypi homebridge[9407]:   Duration: N/A, start: 1.200000, bitrate: N/A
May 01 19:49:11 raspberrypi homebridge[9407]:     Stream #0:0, 46, 1/90000: Video: h264 (Main), 1 reference frame, yuvj420p(pc, bt709, progressive, left), 640x360 (640x368) [SAR 189:190 DAR 168:95], 0/1, 20 fps, 20 tbr, 90k tbn, 40 tbc
May 01 19:49:11 raspberrypi homebridge[9407]: Successfully opened the file.
May 01 19:49:11 raspberrypi homebridge[9407]: Parsing a group of options: output url srtp://192.168.178.101:53880?rtcpport=53880&localrtcpport=53880&pkt_size=1316.
May 01 19:49:11 raspberrypi homebridge[9407]: Applying option map (set input stream mapping) with argument 0:0.
May 01 19:49:11 raspberrypi homebridge[9407]: Applying option vcodec (force video codec ('copy' to copy stream)) with argument copy.
May 01 19:49:11 raspberrypi homebridge[9407]: Applying option pix_fmt (set pixel format) with argument yuv420p.
May 01 19:49:11 raspberrypi homebridge[9407]: Applying option r (set frame rate (Hz value, fraction or abbreviation)) with argument 10.
May 01 19:49:11 raspberrypi homebridge[9407]: Applying option f (force format) with argument rawvideo.
May 01 19:49:11 raspberrypi homebridge[9407]: Applying option vf (set video filters) with argument scale=1280:720.
May 01 19:49:11 raspberrypi homebridge[9407]: Applying option b:v (video bitrate (please use -b:v)) with argument 299k.
May 01 19:49:11 raspberrypi homebridge[9407]: Applying option f (force format) with argument rtp.
May 01 19:49:11 raspberrypi homebridge[9407]: Successfully parsed a group of options.
May 01 19:49:11 raspberrypi homebridge[9407]: Opening an output file: srtp://192.168.178.101:53880?rtcpport=53880&localrtcpport=53880&pkt_size=1316.
May 01 19:49:11 raspberrypi homebridge[9407]: Filtergraph 'scale=1280:720' was defined for video output stream 0:0 but codec copy was selected.
May 01 19:49:11 raspberrypi homebridge[9407]: Filtering and streamcopy cannot be used together.
May 01 19:49:11 raspberrypi homebridge[9407]: [5/1/2020, 19:49:11] [Camera-ffmpeg] ERROR: FFmpeg exited with code 1
```

**Homebridge Config:**

```json
{
	"bridge": {
		"name": "Homebridge-Stbl",
		"username": "XX:XXX:XX:XX:XX:XX",
		"port": XXXXXX,
		"pin": "XXX-XX-XXX"
	},
	"description": "External Cam",
	"platforms": [{
			"platform": "Camera-ffmpeg",
			"cameras": [{
					"name": "XXXXX",
					"manufacturer": "XXXXXX",
					"model": "XXXXXX",
					"motion": true,
					"videoConfig": {
						"source": "-rtsp_transport tcp -re -i rtsp://XXXX@XXX.XXX.XXX.XXX:XXX/Streaming/Unicast/channels/XXX",
						"maxStreams": 3,
						"maxWidth": 1920,
						"maxHeight": 1080,
						"maxFPS": 25,
						"audio": false,
						"vcodec": "copy"
					}
				},
				{
					"name": "XXXXX",
					"manufacturer": "XXXXXX",
					"model": "XXXXXX",
					"motion": true,
					"videoConfig": {
						"source": "--rtsp_transport tcp -re -i rtsp://XXXX@XXX.XXX.XXX.XXX:XXX/Streaming/Unicast/channels/XXX",
						"maxStreams": 3,
						"maxWidth": 1920,
						"maxHeight": 1080,
						"maxFPS": 25,
						"audio": false,
						"vcodec": "copy"
					}
				},
				{
					"name": "XXXXX",
					"manufacturer": "XXXXXX",
					"model": "XXXXXX",
					"motion": true,
					"videoConfig": {
						"source": "-rtsp_transport tcp -re -i rtsp://XXXX@XXX.XXX.XXX.XXX:XXX/Streaming/Unicast/channels/XXX",
						"maxStreams": 3,
						"maxWidth": 1920,
						"maxHeight": 1080,
						"maxFPS": 25,
						"audio": false,
						"vcodec": "copy"
					}
				},
				{
					"name": "XXXXX",
					"manufacturer": "XXXXXX",
					"model": "XXXXXX",
					"motion": true,
					"videoConfig": {
						"source": "-rtsp_transport tcp -re -i rtsp://XXXX@XXX.XXX.XXX.XXX:XXX/Streaming/Unicast/channels/XXX",
						"maxStreams": 3,
						"maxWidth": 1920,
						"maxHeight": 1080,
						"maxFPS": 25,
						"audio": false,
						"vcodec": "copy"
					}
				}
			]
```

**Screenshots:**

**Environment:**

* **Node.js Version**: 
v12.14.0
* **NPM Version**: 
6.14.4
* **Homebridge Version**: 
1.0.4
* **Homebridge Camera FFmpeg Version**:
ffmpeg version 4.1.4-1+rpt1~deb10u1
* **Operating System**:  Raspbian

**Comment by lukehoersten** [comment]
Any known workaround for this until a fix hits npm?

**Comment by donavanbecker** [comment]
@lukehoersten I changed the codecs from copy to `libx264 `, for the time being.

**Comment by mrMiimo** [comment]
> Any known workaround for this until a fix hits npm?

You can restore the previous version 0.1.17 using the command:         “sudo npm install https://github.com/KhaosT/homebridge-camera-ffmpeg#” or something like that...

**Comment by mrMiimo** [comment]
> Any known workaround for this until a fix hits npm?

You can restore the previous version 0.1.17 using the command:         “sudo npm install https://github.com/KhaosT/homebridge-camera-ffmpeg#” or something like that...
Or replace ffmpeg.js witch this version https://github.com/KhaosT/homebridge-camera-ffmpeg/blob/9e96dacd65fc6fc38d32ad66d73b99f8934aa4c1/ffmpeg.js <— i made this and works

**Comment by donavanbecker** [comment]
This is fixed with #475

**Comment by mrMiimo** [comment]
Thanks 🙋🏻‍♂️
